Linux_kernel_diagram.dot

```diff
@@ -553,7 +553,7 @@ digraph "Linux_kernel_diagram" {
 		RAM;
 	}
 	bottom [
-		label = "© 2007-2021 Constantine Shulyupin http://www.MakeLinux.net/kernel/diagram",
+		label = "© 2007 Constantine Shulyupin https://makelinux.github.io/kernel/diagram",
```

Comment:
Just wanted to point out why I removed the second year in the copyright notice:

It is deemed unnecessary to update the date(s) in those notices as explained in the [FAQs of the REUSE project](https://reuse.software/faq/). (Nice read for software authors, btw!)